src/shared/context/userContext.jsx

```
@@ -0,0 +1,22 @@
+//TODO: 유저 정보 저장 컨텍스트 구현
+
+import React, { createContext, useContext, useState } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)